src/common/Smi.Common/Messages/DicomFileMessage.cs

```diff
@@ -119,8 +111,7 @@ public override string ToString()
 
         private bool Equals(DicomFileMessage other)
         {
-            return
-                string.Equals(NationalPACSAccessionNumber, other.NationalPACSAccessionNumber)
+            return true
```

Comment:
why "true" why not just remove the leading &&

Reply:
This is also the '1 for Constant condition' lgtm issue

Reply:
It's an Incredibly minor style choice, so all the conditionals on the following lines line up nicely. I.e.

```csharp
return
    Foo()
    && Bar()
    ...;
```

or

```csharp
return Foo()
    && Bar()
    ...;
```

compared to

```csharp
return true
    && Foo()
    && Bar()
    ...;
```

Reply:
Ok... I would just use whitespace personally.  The true technically introcues 1 more machine OP although the precompiler probably refactores it away during building.  But it's your code.

```
return
       Foo()
    && Bar()
    ...;
```